apps/admin_panel/assets/src/components/Alerter.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const Alerter = ({ alert }) => (
-  <div>
+  <div className="alert-center">
```

Comment:
Not sure about the best practice but maybe move `alert-center` to stay with `alert alert-center ${alert.type}` to keep the markup clean?

Reply:
The result is different because the div area is different

![image](https://user-images.githubusercontent.com/4509570/37756008-5d309ed2-2dda-11e8-8635-f5391ace888f.png)